### PR TITLE
Calculate results

### PR DESCRIPTION
**Pull request by jakubste**

### CODE REVIEW

File: ecs/elections/exceptions.py [hunk 1]
```diff
@@ -15,12 +15,12 @@ def __str__(self):
 
 class IncorrectPreferenceOrderException(Exception):
     def __str__(self):
-        return "Error of votes' preference order"
+        return "Error of votes preference order"
```
Comment (kasptom) [inline ecs/elections/exceptions.py]: "voter's" maybe?

Comment (jakubste) [inline ecs/elections/exceptions.py]: I have problems with apostrof sign in tests, so I just deleted them… :wink: Doesn't matter that much

Comment (kasptom) [inline ecs/elections/exceptions.py]: but do we talk about voter or vote?

Comment (jakubste) [inline ecs/elections/exceptions.py]: Oh, sorry, missed that. Voter, definitely.

Comment (jakubste) [inline ecs/elections/exceptions.py]: On the other hand… This is about .soc file, so it could be vote :dancer:

Comment (kasptom) [inline ecs/elections/exceptions.py]: but preference is rather the trait of a voter and can be used imho as a synonim of vote (as the sequence of candidates)

Comment (jakubste) [inline ecs/elections/exceptions.py]: @DanOgiela wrote these originally, maybe he can elaborate :smile:

**Comment by DanOgiela** [comment]
I think both terms are correct ;)

**Pull request merged**